PowerEditor/src/MISC/Common/Common.h

```diff
@@ -83,12 +82,11 @@ const bool dirDown = false;
 	#define generic_strtok strtok
 	#define generic_strftime strftime
 	#define generic_fprintf fprintf
-	#define generic_sprintf sprintf
+	#define generic_sprintf snprintf
 	#define generic_sscanf sscanf
 	#define generic_fopen fopen
 	#define generic_fgets fgets
 	#define generic_stat _stat
-	#define generic_sprintf sprintf
 	#define COPYDATA_FILENAMES COPYDATA_FILENAMESA
 #endif

```

Comment:
> generic_sprintf was defined twice for ansi and unicode, just removed one of it first unicode version is unchanged and therefore not [visible] here,

Ahh, ok.

> maybe the ansi block could be skipped completely as just unicode is supported since some time

Well, what we should REALLY do is just use [the generic mappings](https://msdn.microsoft.com/en-us/library/c426s321.aspx) that [Microsoft provides](https://msdn.microsoft.com/en-us/library/tsbaswba.aspx) instead of trying to roll our own..... it's really silly NOT to, since we're ***already*** using `TCHAR`!